### ISSUE TITLE
Inter : font-weight stopped working in Firefox

### ISSUE THREAD

**Issue by SchlesserClaude** [google/fonts#5172]
Specify the font name in title with a short description of the bug. 
Please report any issue related to Noto fonts in this repository: https://github.com/googlefonts/noto-fonts/issues.

**Describe the bug**
font-weight stopped working for "Inter" in Firefox since today, no changes to website.

**To Reproduce**
@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&family=Source+Code+Pro:wght@500&display=swap');

span {
font-family: "Inter";
font-size: 14px;
font-weight: 600;
}

<span>test</span>

Open in Firefox, open inspector and disable/change the font-weight.

Steps to reproduce the behavior.

**Expected behavior**
font-weight working for Inter in Firefox.

**Screenshots**
![inter-chrome](https://user-images.githubusercontent.com/1052414/187972261-d9070e6b-d2f1-4465-a29e-06ed7c1e92d0.png)
![inter_firefox](https://user-images.githubusercontent.com/1052414/187972377-2929f912-ec76-47db-8189-a7cf2dbaa822.png)

**Additional context**
Add any other context about the problem here (OS version, browser, printer…)

**Comment by RosaWagner** [comment]
@SchlesserClaude Is it still happening?

**Comment by SchlesserClaude** [comment]
Yes, Google seems to do some user agent recognition. If you change the user agent, then it works again. We have now started hosting the fonts locally using https://google-webfonts-helper.herokuapp.com/fonts

**Comment by thlinard** [comment]
> We have now started hosting the fonts locally using https://google-webfonts-helper.herokuapp.com/fonts

Which is not a solution either: https://github.com/majodev/google-webfonts-helper/issues/98#issuecomment-723660375

In your case, Inter repository provides fonts in WOFF and WOFF2 formats: https://github.com/rsms/inter/releases
